Teapot/URLResponse+Convenience.swift

```diff
@@ -9,10 +9,6 @@ extension URLResponse {
         return self.asHTTP.statusCode
     }
 
-    var allHeaderFields: [String: String] {
-        return self.asHTTP.allHeaderFields
-    }
-
```

Comment:
Why remove this convenience method, out of curiosity?

Reply:
According to the compiler it calls itself now 👀

Reply:
It does if we define one, as we've done by declaring this one. Some nice 🐔 🥚 nonsense!

Reply:
yeah, in talking this through separately with Igor it looks like it decided to warn that there's already a method with this name on `HTTPURLResponse`. 

FWIW this is part of why I prefer to prefix extension method names when extending apple classes, so it's clearer to the reader and compiler when you're calling an extension method rather than potentially calling something with unclear resolution.

However since this is all already published without that, meh.